examples/esi_example_variants/fastly.toml

```
@@ -0,0 +1,28 @@
+# This file describes a Fastly Compute package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
+authors = ["kailan@enviark.com"]
```

```suggestion
authors = ["kblanks@fastly.com"]
```
I think we usually leaving those empty or with fastly emails, no?